agimus_controller_ros/controller_base.py

```diff
@@ -170,33 +240,65 @@ def solve(self):
         x0 = np.concatenate(
             [sensor_msg.joint_state.position, sensor_msg.joint_state.velocity]
         )
+        self.target_translation_object_to_effector = None
+        self.in_world_M_effector = None
         self.fill_buffer()
         if self.traj_buffer.get_size(self.point_attributes) > 0:
             point = self.traj_buffer.get_points(1, self.point_attributes)[0]
             self.last_point = point
+            new_x_ref = point.get_x_as_q_v()
```

Comment:
why?

Reply:
I did that because I wanted to store trajectory point in my plot data only if we were extracting new points but now I'm thinking it would also be fine to continue store the point even if the buffer is empty

Reply:
ok fine keep this I haven't read the code properly